<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>editDialog</class>
    <widget class="QDialog" name="editDialog">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>341</width>
                <height>458</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Edit accounts</string>
        </property>
        <widget class="QTableWidget" name="tableWidget">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>90</y>
                    <width>321</width>
                    <height>361</height>
                </rect>
            </property>
            <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="showGrid">
                <bool>true</bool>
            </property>
            <property name="wordWrap">
                <bool>true</bool>
            </property>
            <property name="cornerButtonEnabled">
                <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
                <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
                <bool>false</bool>
            </attribute>
            <column>
                <property name="text">
                    <string>Username</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>Password</string>
                </property>
            </column>
        </widget>
        <widget class="QPushButton" name="btnAdd">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>75</width>
                    <height>24</height>
                </rect>
            </property>
            <property name="text">
                <string>Add</string>
            </property>
        </widget>
        <widget class="QPushButton" name="btnRm">
            <property name="geometry">
                <rect>
                    <x>90</x>
                    <y>10</y>
                    <width>75</width>
                    <height>24</height>
                </rect>
            </property>
            <property name="text">
                <string>Remove</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="inputUn">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>60</y>
                    <width>113</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="labelUn">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>40</y>
                    <width>71</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Username:</string>
            </property>
        </widget>
        <widget class="QLabel" name="labelPw">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>40</y>
                    <width>71</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Password:</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="inputPw">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>60</y>
                    <width>113</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QDialogButtonBox" name="buttonBox">
            <property name="geometry">
                <rect>
                    <x>260</x>
                    <y>10</y>
                    <width>71</width>
                    <height>71</height>
                </rect>
            </property>
            <property name="orientation">
                <enum>Qt::Vertical</enum>
            </property>
            <property name="standardButtons">
                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
            <property name="centerButtons">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QPushButton" name="btnSvts">
            <property name="geometry">
                <rect>
                    <x>170</x>
                    <y>10</y>
                    <width>75</width>
                    <height>24</height>
                </rect>
            </property>
            <property name="text">
                <string>Save this</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
